backend/src/test/java/backend/mulkkam/intake/service/IntakeAmountServiceIntegrationTest.java

```diff
@@ -96,6 +103,41 @@ void error_memberIsNotExisted() {
                     () -> intakeAmountService.modifyTarget(intakeTargetAmountModifyRequest, Long.MAX_VALUE));
             assertThat(exception.getErrorCode()).isEqualTo(NotFoundErrorCode.NOT_FOUND_MEMBER);
         }
+
+        @DisplayName("해당 수정이 추천에 의한 수정일 경우 금일 목표에만 반영된다")
+        @Test
+        void success_recommendAmount() {
+            // given
+            int memberTargetAmount = 1500;
+            Member member = MemberFixtureBuilder
+                    .builder()
+                    .targetAmount(new Amount(memberTargetAmount))
+                    .build();
+            memberRepository.save(member);
+            IntakeHistory intakeHistory = IntakeHistoryFixtureBuilder
+                    .withMember(member)
+                    .date(LocalDate.now())
+                    .build();
+            intakeHistoryRepository.save(intakeHistory);
+            ModifyIntakeTargetAmountByRecommendRequest modifyIntakeTargetAmountByRecommendRequest = new ModifyIntakeTargetAmountByRecommendRequest
+                    (
+                            1000
+                    );
+            // when
+            intakeAmountService.modifyTargetByRecommended(modifyIntakeTargetAmountByRecommendRequest, member.getId());
+            Optional<IntakeHistory> findIntakeHistory = intakeHistoryRepository.findByMemberIdAndHistoryDate(
+                    member.getId(),
+                    LocalDate.now());
+            Optional<Member> findMember = memberRepository.findById(member.getId());
+
+            // then
+            assertSoftly(softly -> {
+                softly.assertThat(findIntakeHistory).isPresent();
+                softly.assertThat(findIntakeHistory.get().getTargetAmount().value()).isEqualTo(1000);
+                softly.assertThat(findMember).isPresent();
```

Comment:
```suggestion
                softly.assertThat(findIntakeHistory.get().getTargetAmount()).isEqualTo(new Amount(1000));
```
이렇게 VO로 검증하는 거는 어떻게 생각하세용